src/repositories/readingtip_repository.py

```diff
@@ -8,6 +8,10 @@ def __init__(self):
     def get_tips(self, user):
         return ReadingTip.query.filter_by(user=user).all()
 
+    def update_tip(self, tip_id, title, link):
+        db.session.query(ReadingTip).filter_by(id=tip_id).update({"title": title, "link": link})
```

Comment:
Should we stick to `ReadingTip.query` instead of `db.session.query(ReadingTip)`?